docs_src/components/Home.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+  import { link } from 'svelte-spa-router';
+
+  import { components } from '../routes/components';
+</script>
+
+<h1>Components</h1>
+
+<div class="grid-width--small">
+  <p>Components mainly consists of svelte components, but must can be used as straight HTML components</p>
```

Comment:
Kunne ændres til nedenstående, hvis den er korrekt forstået.
"Components mainly consists of svelte components but can be used as straight HTML components if it's a must"